platform/ios/export/export_plugin.cpp

```diff
@@ -2260,6 +2421,82 @@ Error EditorExportPlatformIOS::_export_project_helper(const Ref<EditorExportPres
 		return ERR_FILE_NOT_FOUND;
 	}
 
+	// HACK: We don't want to run the simulator library generation code anymore after GH-102179, but removing it
+	// triggers an internal compiler error with latest mingw-gcc... For now we bring it back as a workaround
+	// with a condition that should never be true (that project setting doesn't exist).
+
+	// Check and generate missing ARM64 simulator library.
+	if (ProjectSettings::get_singleton()->has_setting("ios_generate_simulator_library_if_missing")) {
```

Comment:
Do we have to use this exact string? Or could we use a dummy string to make it clear this setting doesn't exist? Something like "dummy_setting"

Reply:
It can be any name. I think it should be complex enough so that users don't define it by mistake, but yeah I can make it more dummy sounding.

Reply:
"this_is_a_dummy_setting_do_not_use"

Reply:
That's maybe better than what I just amended :P